client/components/home/Status.vue

```diff
@@ -0,0 +1,114 @@
+        :headers="headerDownload"
+        :items="[step]"
+        :items-per-page="1"
+        class="elevation-1"
+        hide-default-footer
+      >
+        <template #[`item.status`]="{item}">
+          <div v-if="item.status === 'success'">
+            <v-icon class="text-center" size="30" color="green darken-2">
+              mdi-check
+            </v-icon>
+          </div>
+          <div v-else>
+            <v-progress-circular
+              :size="70"
+              :width="7"
+              indeterminate
+              color="green"
+            />
+          </div>
+        </template>
+
+        <template #[`item.percent`]="{item}">
+          {{ item.percent }}%
+        </template>
+        <template #[`item.took`]="{item}">
+          {{ item.took }} sec
+        </template>
+      </v-data-table>
+
+      <v-data-table
+        v-if="step.task === 'insert'"
+        :headers="headerInsert"
+        :items="[step]"
+        :items-per-page="1"
+        class="elevation-1"
+        hide-default-footer
+      >
+        <template #[`item.status`]="{item}">
+          <div v-if="item.status === 'success'">
+            <v-icon class="text-center" size="30" color="green darken-2">
+              mdi-check
+            </v-icon>
+          </div>
+          <div v-else>
+            <v-progress-circular
+              :size="30"
+              :width="3"
+              indeterminate
+              color="green"
+            />
+          </div>
+        </template>
+        <template #[`item.percent`]="{item}">
+          {{ item.percent }}%
+        </template>
+        <template #[`item.took`]="{item}">
+          {{ item.took }} sec
+        </template>
+      </v-data-table>
+    </v-card-text>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    status: {
+      type: Object,
+      default: () => {}
+    }
+  },
+  data: () => {
+    return {}
+  },
+  computed: {
+    headerInsert () {
+      return [
+        { text: this.$t('ui.components.home.Status.task'), value: 'task' },
+        { text: this.$t('ui.components.home.Status.took'), value: 'took' },
+        { text: this.$t('ui.components.home.Status.percent'), value: 'percent' },
+        { text: this.$t('ui.components.home.Status.file'), value: 'file' },
+        { text: this.$t('ui.components.home.Status.linesRead'), value: 'linesRead' },
+        { text: this.$t('ui.components.home.Status.status'), value: 'status' }
+      ]
+    },
+    headerDownload () {
+      return [
+        { text: this.$t('ui.components.home.Status.task'), value: 'task' },
+        { text: this.$t('ui.components.home.Status.took'), value: 'took' },
+        { text: this.$t('ui.components.home.Status.percent'), value: 'percent' },
+        { text: this.$t('ui.components.home.Status.file'), value: 'file' },
+        { text: this.$t('ui.components.home.Status.status'), value: 'status' }
+      ]
+    }
+  },
+  methods: {
+    setIcon (status) {
```

Comment:
I guess this method is not used anymore (and should rather be a computed property anyway)